src/gui/search/searchjobwidget.cpp

```diff
@@ -394,8 +395,10 @@ void SearchJobWidget::contextMenuEvent(QContextMenuEvent *event)
     auto *menu = new QMenu(this);
     menu->setAttribute(Qt::WA_DeleteOnClose);
 
-    menu->addAction(UIThemeManager::instance()->getIcon("download"), tr("Download")
-        , this, &SearchJobWidget::downloadTorrents);
+    menu->addAction(UIThemeManager::instance()->getIcon("download"), tr("Open download window")
+        , this, [this]() { downloadTorrents(ShowAddNewTorrentDialog::Yes); });
+    menu->addAction(UIThemeManager::instance()->getIcon("downloading"), tr("Start downloading")
```

Comment:
"Start downloading" -> I will just call it "Download".

Reply:
It should use the same icon `download` like the other.

Reply:
Fixed.